### DISCUSSION TITLE
Battery drain in deep sleep

### DISCUSSION THREAD

**Discussion by cascafico** [markqvist/Reticulum#592] [Help & Questions] [answered]
I'm testing a Heltec v3 with 4Ah battery.
AFAIK, to set device in "deep sleep" mode I usually keep P button pressed, then R button, then release. Display goes off.

It happens that battery get discharged in approx 48h.

Previously same device, same battery with Meshtastic firmware could keep charge for weeks .

Is the switch off method wrong?

**Comment by markqvist** [comment, marked as answer]
The button sequence you describe here is going to put the device into bootloader mode, not sleep mode. The screen will be off, but everything else is on, and the MCU is running at full power.

To put the device into deep sleep, simply hold the user button for a second and release.
